example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -51,7 +63,7 @@ struct PrintBufferKernel
         for(size_t i(linearizedGlobalThreadIdx[0]); i < extents.prod(); i += globalThreadExtent.prod())
         {
             // NOTE: hard-coded for unsigned int
-            printf("%u ", buffer[i]);
+            printf("%u:%u ", static_cast<int32_t>(i), buffer[logIdToPitchedId<2>(i,pitch)]);
```

Comment:
```
/home/travis/build/ComputationalRadiationPhysics/alpaka/example/bufferCopy/src/bufferCopy.cpp:66:19: error: format '%u' expects argument of type 'unsigned int', but argument 2 has type 'int32_t {aka int}' [-Werror=format=]
             printf("%u:%u ", static_cast<int32_t>(i), buffer[logIdToPitchedId<2>(i,pitch)]);
```
You might need a `static_cast<int32_t>` for the second parameter as well.

Reply:
I thought about this, but didn't want to fix too much. Never touch a running system. :roll_eyes:  At least for little endian it works well this way. :stuck_out_tongue:
jk, I will change this, too.